test/sequential/maintenance-pallet.api.rolldown.test.ts

```diff
@@ -236,7 +236,13 @@ describe.each(["mm", "upgradabilityMm"])(
         "%s operations are allowed in mm",
         async (testName) => {
           testLog.getLog().info("DEBUG::TestName - " + testName);
-          const [extrinsic, signer] = tests[testName];
+          let signer: User;
+          const [extrinsic, testsSigner] = tests[testName];
+          if (mmMode === "mm") {
+            signer = testsSigner;
+          } else {
+            signer = users[3];
+          }
```

Comment:
why not simply creating a new user, this looks quite strange. isn't it?

Reply:
I decided that if we have an unused account, we can use it here. But yes, we can create a new user. Added